schemas/v3.0/schema.yaml

```diff
@@ -763,6 +783,9 @@ definitions:
         type: object
         additionalProperties:
           $ref: '#/definitions/MediaType'
+        patternProperties:
+          '\*\/\*|^application/x-www-form-urlencoded$|^multipart\/.+$':
```

Comment:
Should `*/*` have `^`/`$` anchors?
Should `application/*` be matched, as that would apply to `application/x-www-form-urlencoded`?

Reply:
what is the purpose of the `/s` anchors?  Why would it be needed? 

`application/*` would match any `application` media type and that's _not_ what we want here., although that would enable quite a few different media types to allow `encoding` schema.   Matching `*/*` was only for backwards compatibility per Darrell's comment.